src/test/java/org/fundacionjala/pivotal/cucumber/steps/CommonSteps.java

```diff
@@ -0,0 +1,22 @@
+package org.fundacionjala.pivotal.cucumber.steps;
+
+import cucumber.api.java.en.Given;
+import org.fundacionjala.core.util.Environment;
+import org.fundacionjala.pivotal.pageobjects.dashboard.Dashboard;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.fundacionjala.pivotal.pageobjects.dashboard.Dashboard'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)